### PR TITLE
Template fixes and improvements

### PR DESCRIPTION
**Pull request by gber**

### CODE REVIEW

File: personal/privacyidea/default/mfatype_webauthn/phase01.tpl [hunk 1]
```diff
@@ -21,22 +21,40 @@
...
+                <li>{t}Confirm the setup using a device-specific mechanism, e.g. by touching a sensor on the key.{/t}</li>
+                <li>{t}The browser will proceed to the next page if the setup has been sucessful.{/t}</li>
+            </ol>
+            <button class="btn primary"
+                name="mfa_webauthn_confirm"
+                type="submit">{t}Start setup{/t}</button>
+        </div>
 
-<div class="card-paneli red red-lighten4 text-red text-darken-4" id="mfaWebAuthnError"></div>
+        <div class="card-panel red lighten-4 red-text text-darken-4" id="mfaWebAuthnError"></div>
```
Comment (dzatoah) [inline personal/privacyidea/default/mfatype_webauthn/phase01.tpl:41]: Why do we need a message box here anyway?

File: personal/privacyidea/default/mfatype_paper/phase01.tpl [hunk 2]
```diff
@@ -25,10 +25,26 @@
...
+        </video>
+        <p xmlns:cc="http://creativecommons.org/ns#" style="text-align: right">&copy; 2023
+            <span property="cc:attributionName">Thomas Häpp, Universität Bonn</span>,
+            licensed under <a
+            href="http://creativecommons.org/licenses/by-sa/4.0/?ref=chooser-v1"
+            target="_blank" rel="license noopener noreferrer"
+            style="display:inline-block;">CC BY-SA 4.0<img style="height:22px!important;margin-left:3px;vertical-align:text-bottom;"
+            src="plugins/privacyidea/images/cc.svg"><img style="height:22px!important;margin-left:3px;vertical-align:text-bottom;"
+            src="plugins/privacyidea/images/by.svg"><img style="height:22px!important;margin-left:3px;vertical-align:text-bottom;"
+            src="plugins/privacyidea/images/sa.svg"></a></p>
```
Comment (dzatoah) [inline personal/privacyidea/default/mfatype_paper/phase01.tpl:46]:
```suggestion
            style="display:inline-block;">CC BY-SA 4.0<img style="height:22px!important;margin-left:3px;vertical-align:text-bottom;"
            src="images/cc.svg"><img style="height:22px!important;margin-left:3px;vertical-align:text-bottom;"
            src="images/by.svg"><img style="height:22px!important;margin-left:3px;vertical-align:text-bottom;"
            src="images/sa.svg"></a></p>
```

File: personal/privacyidea/default/mfatype_totp/phase01.tpl [hunk 3]
```diff
@@ -18,32 +18,48 @@
...
-    <button class="btn primary"
-        name="mfa_generate_totp_secret"
-        type="submit">{t}Generate new QR code{/t}
-    </button>
-    {* Make sure to provide needed variables again. *}
-    <input type="hidden" id="tokenDescription" name="tokenDescription" value="{$tokenDescription}">
-    <input type="hidden" id="tokenSerial" name="tokenSerial" value="{$tokenSerial}">
+    <div class="col m12 l8" style="background-color: #dfdfdf; text-align:center;">
+        <video class="responsive-video" autoplay loop muted>
+            <source src="plugins/privacyidea/videos/totp.mp4" type="video/mp4">
```
Comment (dzatoah) [inline personal/privacyidea/default/mfatype_totp/phase01.tpl:51]:
```suggestion
            <source src="videos/privacyidea-totp.mp4" type="video/mp4">
```

File: personal/privacyidea/default/mfatype_totp/phase01.tpl [hunk 4]
```diff
@@ -18,32 +18,48 @@
...
+        </video>
+        <p xmlns:cc="http://creativecommons.org/ns#" style="text-align: right">&copy; 2023
+            <span property="cc:attributionName">Thomas Häpp, Universität Bonn</span>,
+            licensed under <a
+            href="http://creativecommons.org/licenses/by-sa/4.0/?ref=chooser-v1"
+            target="_blank" rel="license noopener noreferrer"
+            style="display:inline-block;">CC BY-SA 4.0<img style="height:22px!important;margin-left:3px;vertical-align:text-bottom;"
+            src="plugins/privacyidea/images/cc.svg"><img style="height:22px!important;margin-left:3px;vertical-align:text-bottom;"
+            src="plugins/privacyidea/images/by.svg"><img style="height:22px!important;margin-left:3px;vertical-align:text-bottom;"
+            src="plugins/privacyidea/images/sa.svg"></a></p>
```
Comment (dzatoah) [inline personal/privacyidea/default/mfatype_totp/phase01.tpl:61]:
```suggestion
            style="display:inline-block;">CC BY-SA 4.0<img style="height:22px!important;margin-left:3px;vertical-align:text-bottom;"
            src="images/cc.svg"><img style="height:22px!important;margin-left:3px;vertical-align:text-bottom;"
            src="images/by.svg"><img style="height:22px!important;margin-left:3px;vertical-align:text-bottom;"
            src="images/sa.svg"></a></p>
```

File: personal/privacyidea/default/mfatype_webauthn/phase01.tpl [hunk 5]
```diff
@@ -21,22 +21,40 @@
...
+    </div>
+    <div class="col m12 l6" style="background-color: #dfdfdf; text-align:center;">
+        <video class="responsive-video" autoplay loop muted>
+            <source src="plugins/privacyidea/videos/fido.mp4" type="video/mp4">
```
Comment (dzatoah) [inline personal/privacyidea/default/mfatype_webauthn/phase01.tpl:45]:
```suggestion
            <source src="videos/privacyidea-fido.mp4" type="video/mp4">
```

File: personal/privacyidea/default/mfatype_webauthn/phase01.tpl [hunk 6]
```diff
@@ -21,22 +21,40 @@
...
+        </video>
+        <p xmlns:cc="http://creativecommons.org/ns#" style="text-align: right">&copy; 2023
+            <span property="cc:attributionName">Thomas Häpp, Universität Bonn</span>,
+            licensed under <a
+            href="http://creativecommons.org/licenses/by-sa/4.0/?ref=chooser-v1"
+            target="_blank" rel="license noopener noreferrer"
+            style="display:inline-block;">CC BY-SA 4.0<img style="height:22px!important;margin-left:3px;vertical-align:text-bottom;"
+            src="plugins/privacyidea/images/cc.svg"><img style="height:22px!important;margin-left:3px;vertical-align:text-bottom;"
+            src="plugins/privacyidea/images/by.svg"><img style="height:22px!important;margin-left:3px;vertical-align:text-bottom;"
+            src="plugins/privacyidea/images/sa.svg"></a></p>
```
Comment (dzatoah) [inline personal/privacyidea/default/mfatype_webauthn/phase01.tpl:55]:
```suggestion
            style="display:inline-block;">CC BY-SA 4.0<img style="height:22px!important;margin-left:3px;vertical-align:text-bottom;"
            src="images/cc.svg"><img style="height:22px!important;margin-left:3px;vertical-align:text-bottom;"
            src="images/by.svg"><img style="height:22px!important;margin-left:3px;vertical-align:text-bottom;"
            src="images/sa.svg"></a></p>
```

**Review by dzatoah — COMMENTED** [review]
Please rename the videos to privacyidea-*.mp4 and alter the src-path.

Thanks :)

File: personal/privacyidea/default/mfatype_webauthn/phase01.tpl (hunk 1 above)
Comment (gber) [reply personal/privacyidea/default/mfatype_webauthn/phase01.tpl:41]: It's in the the mockup: errors from Webauthn setup are supposed to be displayed here, if an error occurs the button should restart the setup.

**Comment by gber** [comment]
> Please rename the videos to privacyidea-*.mp4 and alter the src-path.
> 
> Thanks :)

Are you sure about this? All other plugins seem to place their static files namespaced under  `<gosa path>/html/plugins/<name>/`?

**Comment by sunweaver** [comment]
> > Please rename the videos to privacyidea-*.mp4 and alter the src-path.
> > Thanks :)
> 
> Are you sure about this? All other plugins seem to place their static files namespaced under `<gosa path>/html/plugins/<name>/`?

I agree with Guido here: html/videos/*.mp4 should be our paths...

**Comment by dzatoah** [comment]
Ah yes, you're right. Thanks. I'll merge now :)

File: personal/privacyidea/default/mfatype_webauthn/phase01.tpl (hunk 1 above)
Comment (dzatoah) [reply personal/privacyidea/default/mfatype_webauthn/phase01.tpl:41]: Ah okay.

**Pull request merged**